### PR TITLE
Drop feedback

### PR DESCRIPTION
**Pull request by vivieatto**

### CODE REVIEW

File: Main/GNWMap.pde [hunk 1]
```diff
@@ -144,127 +122,4 @@ class GNWMap
     Building newBuilding = new Building(name, c, doorNodeId, x1, y1, x2, y2, x3, y3, x4, y4);
     buildings.put(name, newBuilding);
   }
-  
-
```
Comment (IrisChu) [inline Main/GNWMap.pde]: Do we really want to remove these lines?

**Comment by IrisChu** [comment]
@vivieatto  Hey Jessi, I took a look at the logic for implementing the highlight, and it looks good! But I noticed that the drop feedback branch wasn't branched out from master. Sometimes that's ok, like if we know that the src branch is up-to-date (i.e. not behind any commits compared to master) and will definitely be merged. But Ling haven't made a pull request for her branch yet (@susu880229  please make a pull request when you are ready) and it's currently 16 commits behind master. So we'll have to postpone approving this pull request until that branch has successfully merged into master first. Otherwise, if we approve of this branch first, some of the commits from her branch would also be merged into master (and we don't want the mix up since it'll probably cause even more merge conflicts that's not easy to resolve).

I'm not as familiar with the flow implementation as either of you, but I also noticed that quite a bit of Ling's implementations were removed in the "drop feedback and change icon position to be above the user's finger.. " commit. I made a direct comment in the area of concern (it is under GNWMap.pde). How is the matrix implemented? I thought it was the createUseFlow() method? If it is, we need to keep that. You might want to consider making a new branch from master and moving your changes manually to that new branch. Or you can keep the same branch and just add all of that stuff back in. 

Sorry for the long post, but I also want to mention that the highlight images could be in higher resolution. The edges looks kinda rough right now. This definitely doesn't have to be done this sprint, cause images can be replaced later on. Just wanted to mention it now before I forget :P

**Comment by vivieatto** [comment]
@IrisChu hahahhaha sorry, it's a github n00b mistake, I accidentally created a new branch out of Ling's branch, but couldn't figure out how to delete it, so I just replaced the Main here with the code that I downloaded from master, that's likely why the commits are all confused. I'll create the new branch on Mon while you're around to make sure it's correct haha!

**Comment by IrisChu** [comment]
@vivieatto  sounds good :)

**Pull request closed without merging**